doc/api/synopsis.md

```diff
@@ -20,8 +20,8 @@ there to show the start of each command.
 Lines that don’t start with `$` or `>` character show the output of the previous
 command.
 
-First, make sure to have downloaded and installed Node.js. See [this guide][]
-for further install information.
+First, make sure to have downloaded and installed Node.js. See
+["Installing Node.js via package manager"][] for further install information.
```

Comment:
Do we usually surround article titles with quotation marks?

Reply:
No, we don't. I'll remove them.